tests/web_api/test_spec.py

```diff
@@ -51,5 +51,5 @@ def test_situation_definition():
     for situation in situation_input, situation_output:
         assert_in('households', dpath.get(situation, '/properties'))
         assert_in('persons', dpath.get(situation, '/properties'))
-        assert_equal("#/definitions/Household", dpath.get(situation, '/properties/households/additionalProperties/$ref'))
-        assert_equal("#/definitions/Person", dpath.get(situation, '/properties/persons/additionalProperties/$ref'))
+        assert "#/definitions/Household", dpath.get(situation == '/properties/households/additionalProperties/$ref')
```

Comment:
Same here.

Reply:
BTW I think Pytest also gives us `assert 'households' in dpath.get(situation, '/properties')`